recipes-kernel/linux/files/bcm2712-raspberrypi5-can.dtso

```diff
@@ -7,29 +7,30 @@
 /plugin/;
 
 &passthrough {
-	can1_osc: can1_osc {
+	can0_osc: mcp251xfd-spi0-0-osc {
```

Comment:
what's the point of changing can0->can1 order? Maybe it worth leaving as is. diff will look neater